Remotion/Web/Development.WebTesting.TestSite/Startup.cs

```diff
@@ -4,11 +4,24 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Remotion.Development.UnitTesting;
+using Remotion.ServiceLocation;
+using Remotion.Web.ExecutionEngine;
 
 namespace Remotion.Web.Development.WebTesting.TestSite
 {
   public class Startup
   {
+    private ServiceLocatorScope _serviceLocatorScope;
+    public Startup ()
+    {
+      //I'm not sure if this is what we need... I think I need some more info on the penultimate list item of the task.
+      var wxeUrlSettings = WxeUrlSettings.Create("~/bin/UrlMapping.xml");
+      var serviceLocator = DefaultServiceLocator.Create();
+      serviceLocator.RegisterSingle(() => wxeUrlSettings);
+      _serviceLocatorScope = new ServiceLocatorScope(serviceLocator);
```

Comment:
the ServiceLocatorScope is only meant for unit tests. (see namespace).

I  recomment we move the UrlMapping.xml to the correct folder so the default works. Otherwise, we initialize the ServiceLocator the same way we do in the ObjectBinding.Web.Test project instead of relying on ServiceLocatorScope.